docs/concepts/api-extension/apiserver-aggregation.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Extending the Kubernetes API with the aggregation layer
+assignees:
+- lavalamp
+- cheftako
+- chenopis
+---
+
+{% capture overview %}
+
+The aggregation layer allows Kubernetes to be extended with additional APIs, beyond what is offered by the core Kubernetes APIs. 
+
+{% endcapture %}
+
+{% capture body %}
+
+## Overview
+
+The aggregation layer enables installing additional Kubernetes-style APIs in your cluster. These can either be pre-built, existing 3rd party solutions, such as [service-catalog](https://github.com/kubernetes-incubator/service-catalog/blob/master/README.md), or user-created APIs like [apiserver-builder](https://github.com/kubernetes-incubator/apiserver-builder/blob/master/README.md), which can get you started.
+
+In 1.7 the aggregation layer runs in-process with the kube-apiserver. Until an extension resource is registered, the aggregation layer will do nothing. Users can add an APIService object to register their API, which “claims” the URL path in the Kubernetes API. At that point, the aggregation layer will proxy anything sent to that API path (e.g. /apis/myextension.mycompany.io/v1/…) to the registered APIService. Ordinarily, the APIService will be implemented by an extension-apiserver in a pod running in the cluster. This extension-apiserver will normally need to be paired with one or more controllers if active management of the added resources is needed. As a result, the apiserver-builder will actually provide a skeleton for both. For example, when the service-catalog is installed, it provides both the extension-apiserver and controller for the services it provides.
```

Comment:
I want to gut check my own understanding of things, so quick question: 

**Is it the case that kube-apiserver is an actual official binary, whereas extension-apiserver is a stand-in name that can refer to *any* roll-your-own custom API server?**

(e.g. I should not expect to be able to literally run `./extension-apiserver` unless I explicitly name the custom API server binary that way?)